apps/server/src/modules/tool/external-tool/controller/api-test/tool.api.spec.ts

```diff
@@ -723,4 +723,58 @@ describe('ToolController (API)', () => {
 			});
 		});
 	});
+
```

Comment:
do you need maybe a describe with test for cases like this:
- when no external Id is given
- when you could not find tool to the toolId

Reply:
without externalId you can't call the endpoint because it is part of the path 😅, but the other is added now :)